Standarized the Python dependencies section for cloud and docker modules (#6118)
wazuh/wazuh-documentation: source/user-manual/agents/cloud-and-container-monitoring.rst

.. Copyright (C) 2015, Wazuh, Inc.

.. meta::
    :description: Learn more about the agent management in Wazuh. In this section, we will show you how to configure an agent to monitor cloud and containers.

.. _cloud_and_container_monitoring:

Cloud and container monitoring
==============================

Wazuh offers the ability to monitor your cloud and container infrastructure, helping you maintain the highest levels of
security. In this section, we will show how to configure your Wazuh agent to monitor various cloud providers and container technologies.

> To learn more about cloud security, check out :doc:`this section </monitoring>`.

Reviewer: ```suggestion
See the :doc:`Cloud security </monitoring>` and :doc:`Container security </container-security/index>` sections to learn more.
```
